provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/exceptions/ProvisioningDeviceClientExceptionManager.java

```diff
@@ -18,38 +18,30 @@ public static void verifyHttpResponse(HttpResponse httpResponse) throws Provisio
     {
         int responseStatus = httpResponse.getStatus();
 
-        String errorMessage = ErrorMessageParser.bestErrorMessage(new String(httpResponse.getErrorReason(), StandardCharsets.UTF_8));
+        byte[] errorReason = httpResponse.getErrorReason();
+
+        String errorMessage = httpResponse.getStatus() + " : " + new String(errorReason, StandardCharsets.UTF_8);
```

Comment:
Previously, we didn't include the status code in this error message, but we should as it helps debug customer issues